scss/_variables.scss

```diff
@@ -731,7 +731,7 @@ $table-cell-padding-x-sm:     .25rem !default;
 $table-cell-vertical-align:   top !default;
 
 $table-color:                 var(--#{$prefix}body-color) !default;
-$table-bg:                    transparent !default;
+$table-bg:                    var(--#{$prefix}body-bg) !default;
```

Comment:
This is a breaking change.

Table background has been transparent since [2012](https://github.com/twbs/bootstrap/commit/d98a2e92340509fe9835103c799d3433c0c0d961#diff-12f64cbedc13e18ede5c900c2ba7f34bc323cf6e44521230bb9a61312dce1483R60)

Also table-primary & table-secondary are available but not table-ternary, why?

@louismaximepiton

Reply:
Confirming the part regarding the table variants. It's mentioned in our [Tables > Variants documentation](https://getbootstrap.com/docs/5.3/content/tables/#variants):

> Heads up! Because of the more complicated CSS used to generate our table variants, they most likely won’t see color mode adaptive styling until v6.